src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -1204,56 +1185,35 @@ static void window_cheats_paint(rct_window* w, rct_drawpixelinfo* dpi)
             colour |= COLOUR_FLAG_INSET;
         }
         int32_t actual_month = _monthSpinnerValue - 1;
-        gfx_draw_string_left(
-            dpi, STR_BOTTOM_TOOLBAR_CASH, gCommonFormatArgs, colour,
-            w->windowPos + ScreenCoordsXY{ XPL(0) + TXTO, YPL(2) + TXTO });
-        gfx_draw_string_left(
-            dpi, STR_YEAR, nullptr, COLOUR_BLACK, w->windowPos + ScreenCoordsXY{ XPL(0) + TXTO, YPL(7) + TXTO });
-        gfx_draw_string_left(
-            dpi, STR_MONTH, nullptr, COLOUR_BLACK, w->windowPos + ScreenCoordsXY{ XPL(0) + TXTO, YPL(8) + TXTO });
-        gfx_draw_string_left(
-            dpi, STR_DAY, nullptr, COLOUR_BLACK, w->windowPos + ScreenCoordsXY{ XPL(0) + TXTO, YPL(9) + TXTO });
-        gfx_draw_string_right(
-            dpi, STR_FORMAT_INTEGER, &_yearSpinnerValue, w->colours[1],
-            w->windowPos + ScreenCoordsXY{ WPL(1) - 34 - TXTO, YPL(7) + TXTO });
+        gfx_draw_string_left(dpi, STR_BOTTOM_TOOLBAR_CASH, gCommonFormatArgs, colour, w->windowPos + ScreenCoordsXY{ 14, 93 });
```

Comment:
I see this number `14` a lot in this PR. There must be some significance to it, so defining a constant would be appropriate IMO.

Reply:
Shall we introduce an `ANCHOR_LCOL_X` = 14 and `ANCHOR_RCOL_X` = 208? They seem to be restricted to `window_cheats_paint`, so I propose we make them local constants there.

Reply:
Local constants would be fine, if it is really unique to the Cheats window.

Reply:
I meant local to the paint function, even, not just the file. Since the value doesn't occur in the same way elsewhere in the file, it makes sense to optimise this locally.